cli/types/cypress.d.ts

```diff
@@ -3019,6 +3019,17 @@ declare namespace Cypress {
      * @see https://on.cypress.io/configuration#experimentalModifyObstructiveThirdPartyCode
      */
     experimentalModifyObstructiveThirdPartyCode: boolean
+    /**
+     * Disables setting document.domain to the document's super domain on injection.
+     * This experiment is to be used for sites that do not work with setting document.domain
+     * due to cross-origin issues. Enabling this option no longer allows for default sub domain
+     * navigations, and will require the use of cy.origin(). This option takes an array of
+     * strings/string globs, and applies defaults for known domains where document.domain is known
```

Comment:
> and applies defaults for known domains where document.domain is known

I think we should drop this portion. I assu,e you mean google, but that's happening today without this enabled.

Reply:
I think I originally assumed the defaults we applied but that context makes sense. removed in dccf967ed9